<?xml version="1.0"?>

<launch>

  <arg name="trial_name" default="default" />

  <node name="em_spcof_name2place" pkg="em_spco_formation" type="em_spcof_name2place.py" output="screen" cwd="node" >
    <param name="trial_name" value="$(arg trial_name)" />
  </node>

</launch>
